### ISSUE TITLE
text-to-video task page

### ISSUE THREAD

**Issue by merveenoyan** [huggingface/hub-docs#789]
Since this task is very trending these days it would be nice to have a task page for that 😊 
You can refer to #369 for a guide!

- [x]  Task variants @snknitin 
- [x]  Use cases @snknitin 
- [x]  Most Used Dataset for the Task @snknitin 
- [x] Most Used model for this task @snknitin 
- [x] Example Spaces @Muskan149
- [ ] Widget model (not possible)
- [x] Task schema @melihyazgan 
- [x] Useful Resources @melihyazgan 
- [x] Metric explanations @tescao  
- [ ] (or any other thing you'd like to add)

**Comment by snknitin** [comment]
Hi, I'd like to help with this

**Comment by Muskan149** [comment]
I'd love to contribute to the cases or/and example spaces section!

**Comment by akashAD98** [comment]
Im also interested

**Comment by snknitin** [comment]
> @snknitin great! which parts would you like to take? you don't have to contribute all of it.

I'd like to take on the top 4 parts

**Comment by melihyazgan** [comment]
Hey hi all,
I would like to help too. Count me in for widget model, task schema, useful resources

**Comment by subi9** [comment]
I would like to contribute as well.

**Comment by gozdeydd** [comment]
I would like to contribute.

**Comment by msdkhairi** [comment]
I would like to contribute as well.

**Comment by summarizeAI** [comment]
Hi, I'd like to contribute to metric explanations.

**Comment by Leumastai** [comment]
Hi, I'd like to contribute to metric explanations.

**Comment by Karenzhang7717** [comment]
I would like to help with any needs for documentation / technical writing. :)

**Comment by Ashish-Soni08** [comment]
Hi Merve! I would also like to contribute.

**Comment by kris-mlguy** [comment]
I'd like to contribute to Most Used model for this task / Example Spaces / Widget model / Task schema.

**Comment by osanseviero** [comment]
Hey all! Thanks for your interest to contribute :exploding_head: this is wild! Just so everyone is aligned

- @snknitin will take a look at  variants, use-cases, datasets and models 
- @Muskan149 example spaces
- widget model is not possible yet as we don't have that
- @tescao  metrics
- @melihyazgan for task schema and useful resources

I updated the top post :hugs: :fire: @akashAD98 @subi9 @elMukaddim  @gozdeydd @msdkhairi  @Ashish-Soni08  once the first wave of PRs is merged feel free to open subsequent PRs with new resources/fixes/docs/etc. Please note that there are possibilities to contribute to all the other task pages as well! :hugs:

**Comment by balugans** [comment]
Would love to be added to this project

**Comment by younver** [comment]
I'd like to contribute if there will be any other open tasks in this issue.

**Comment by ImtiazKhanDS** [comment]
I would like to contribute here @merveenoyan

**Comment by KevKibe** [comment]
I would also like to make a contribution

**Comment by tanersekmen** [comment]
Hi @merveenoyan, I would also like to contribute.

**Comment by ahsanrossi** [comment]
I would like to contribute

**Comment by Adriman2** [comment]
Willing to help!

**Comment by AbdelrahimKoura** [comment]
I would like to contribute

**Comment by AbdelrahimKoura** [comment]
Hi @merveenoyan  I would like to contribute

**Comment by e-olang** [comment]
Hello, I would like to contribute

**Comment by yxzysy** [comment]
I would like to contribute

**Comment by Suraj520** [comment]
Hey @merveenoyan, I'd like to contribute !

**Comment by merveenoyan** [comment]
Hey @Suraj520 @yxzysy @e-olang @AbdelrahimKoura @Adriman2 @ahsanrossi @tanersekmen @KevKibe @ImtiazKhanDS @younver @balugans thanks a lot for the interest! 

Since there's already plenty of people willing to contribute to this one, I opened a separate issue for other tasks you can help with, if this works #792 😊 what do you think?

**Comment by younver** [comment]
> Hey @Suraj520 @yxzysy @e-olang @AbdelrahimKoura @Adriman2 @ahsanrossi @tanersekmen @KevKibe @ImtiazKhanDS @younver @balugans thanks a lot for the interest!
> 
> Since there's already plenty of people willing to contribute to this one, I opened a separate issue for other tasks you can help with, if this works #792 😊 what do you think?

Thank you very much^^

**Comment by Suraj520** [comment]
> Hey @Suraj520 @yxzysy @e-olang @AbdelrahimKoura @Adriman2 @ahsanrossi @tanersekmen @KevKibe @ImtiazKhanDS @younver @balugans thanks a lot for the interest!
> 
> Since there's already plenty of people willing to contribute to this one, I opened a separate issue for other tasks you can help with, if this works #792 blush what do you think?

Sure @merveenoyan  ! :D

**Comment by zehranrgi** [comment]
Hi, I would like to take part in the useful resources. @merveenoyan

**Comment by tauhidshah** [comment]
I would like to help with this

**Comment by merveenoyan** [comment]
@tauhidshah @zehranrgi sure! feel free to open a pull request 😊

**Comment by melihyazgan** [comment]
Hi all,
there is no default folder for text_2_video in main or any other branch or am I missing  something?

**Comment by znym** [comment]
Hi @merveenoyan, I'd also love to contribute, please!

**Comment by merveenoyan** [comment]
@melihyazgan the contributors can copy and paste the `data.ts` in `placeholder` and then create an empty `about.md`.

**Comment by merveenoyan** [comment]
@znym sounds good! feel free to open a PR 🤗

**Comment by melihyazgan** [comment]
> @melihyazgan the contributors can copy and paste the `data.ts` in `placeholder` and then create an empty `about.md` under a folder called `text-to-video`. 

Thanks @merveenoyan

**Comment by snknitin** [comment]
@merveenoyan , @osanseviero , I was sick for a couple of days, so I couldn't get to it till now. I've created a draft pull request for this. Please go through whenever convenient and suggest changes !

**Comment by tescao** [comment]
Hi @merveenoyan @osanseviero I added the metrics section, feel free to review and suggest edits when you can! #881

**Comment by osanseviero** [comment]
Thanks everyone for their contributions. https://huggingface.co/tasks/text-to-video is in :fire:
